WalletWasabi.Gui/Controls/WalletExplorer/WalletActionViewModel.cs

```diff
@@ -30,5 +34,57 @@ public void DisplayActionTab()
 		{
 			IoC.Get<IShell>().AddOrSelectDocument(this);
 		}
+
+		protected void SetWarningMessage(string message)
```

Comment:
It's better than it was before, because it removes code duplication. Toast Notifications are coming almost as fast as winter in Game of Thrones. But they are not here yet.